relayer/relayer.go

```diff
@@ -111,7 +111,7 @@ func (r *Relayer) Start(ctx context.Context) error {
 				}
 
 				// wait for transaction to be mined
-				_, err = r.EVMClient.WaitForTransaction(ctx, ethClient, tx)
+				_, err = r.EVMClient.WaitForTransaction(ctx, ethClient, tx, 15*time.Minute)
```

Comment:
can we make this configurable so users can change this on the fly? perhaps for other timeouts as well, but this one feels more important

Reply:
[93114c2](https://github.com/celestiaorg/orchestrator-relayer/pull/621/commits/93114c2781cec8e24f0df6a304995925a6ed28df)